designs/tripleo/osc-tripleo.md

```diff
@@ -105,6 +119,8 @@ outputs:
                   - {get_param: OSCVolumeSrc}:/opt/vmidc/bin/data/
             environment:
               - OSC_ENV_VAR={get_param: OSCEnvVar}
+              - TripleO::Services::OSC::tls_truststore_file: {get_param:OSCTrustStoreFile}
```

Comment:
I believe anything under `environment` are environment variables for the container. Meaning it should be name=value pair. Are you sure you can define resources like this `TripleO::Services::OSC::tls_truststore_file` here ? Is there a link/ref to another template that does this?

Reply:
I saw this in ODL for example: neutron::plugins::ovs::opendaylight::allowed_network_types: HostAllowedNetworkTypes. The link is:https://github.com/openstack/tripleo-heat-templates/blob/master/puppet/services/opendaylight-ovs.yaml.


Defining like above would enable this parameter to reference from puppet scripts later on (if needed) for example for copying to another location or using rest api's to push this information etc...). I shall add this as TODO item, so that we can clarify.